package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "@shiftcommerce/shift-next",
-  "version": "0.9.2",
+  "version": "0.10.0",
   "description": "",
   "main": "dist/bundle.js",
   "repository": "https://github.com/shiftcommerce/shift-next.git",
+  "engines": {
+    "node": "^11.13.0"
```

Comment:
This is because `Array.prototype.flat()` is only available from Node 11.0.0 onwards.

Reply:
👍